client/util/util.go

```diff
@@ -63,9 +65,51 @@ func LoadSignerBinaryPath(configFilePath string) (path string, err error) {
 	if signerBinaryPath == "" {
 		return "", ErrConfigUnavailable
 	}
+	// Checking for ~ & HOME variables to represent home directory
+	if strings.Contains(signerBinaryPath, "~") || strings.Contains(signerBinaryPath, "HOME") {
+		homeDir, err := os.UserHomeDir()
+		if err != nil {
+			return "", ErrConfigUnavailable
+		}
+		// Updating signerBinaryPath to substitute "~" or "HOME" with the home directory + the rest of the path
+		if strings.Contains(signerBinaryPath, "~") {
+			signerBinaryPath = homeDir + strings.TrimLeft(signerBinaryPath, "~")
+		}
+		if strings.Contains(signerBinaryPath, "HOME") {
+			signerBinaryPath = homeDir + strings.TrimLeft(signerBinaryPath, "HOME")
+		}
+		return signerBinaryPath, nil
+	}
 	return signerBinaryPath, nil
 }
 
+/* ORIGINAL FUNCTION */
```

Comment:
You can remove this commented chunk. We will use git to see the function's history.

Reply:
got it!